volttron/platform/control.py

```diff
@@ -2493,6 +2495,21 @@ def add_parser(*args, **kwargs):
 
     # Parse and expand options
     args = argv[1:]
+
+    # TODO: for auth some of the commands will work when volttron is down and
+    # some will error (example vctl auth serverkey). Do check inside auth
+    # function
+    # Below vctl commands can work even when volttron is not up. For others
+    # volttron need to be up.
+    if args[0] not in ('list', 'tag', 'auth', 'rabbitmq'):
+        # check pid file
+        # TODO: is there a platform level constant file to save 'VOLTTRON_PID'
```

Comment:
Isn't this todo what you add with this pull request?

Reply:
Removed the second TODO.